renderer/canvas.py

```diff
@@ -65,36 +64,55 @@ def _draw_image(self, surface, image_name, position, scale):
         image = pygame.transform.smoothscale(image, scale)
         surface.blit(image, position)
 
-    def _draw_food_image(self, surface, food_name, obs, position):
+    def _draw_item_image(self, surface, item_name, obs, position):
         """
-        Helper to draw a food image on the canvas.
+        Helper to draw a item image on the canvas.
 
         Args:
             surface (pygame.Surface): Surface to draw on
-            food_name (str): Name of the food
+            item_name (str): Name of the item
             obs (List[Literal]): Game state predicates
-            position (np.array): (x, y) position of the food (with pix_square_size factor accounted for)
+            position (np.array): (x, y) position of the item (with pix_square_size factor accounted for)
         """
-        food_image_name = food_name
-        # Check if cut or cooked or fried
-        for literal in obs:        
-            if literal.predicate == "iscut" and literal.variables[0] == food_image_name:
-                food_image_name = "cut" + food_image_name
-            if literal.predicate == "iscooked" and literal.variables[0] == food_image_name:
-                food_image_name = "cooked" + food_image_name
-            if literal.predicate == "isfried":
-                # TODO: This needs to change if we want to have both fried potatoes and french fries
-                if literal.variables[0] == food_image_name:
-                    food_image_name = "fried" + food_image_name
-                elif literal.variables[0] == food_image_name[3:]:
-                    food_image_name = "fried" + food_image_name[3:]
+        item_image_name = item_name
+        
         # Remove and store ID
-        food_id = ""
-        while food_image_name[-1].isdigit():
-            food_id += food_image_name[-1]
-            food_image_name = food_image_name[:-1]
+        item_id = ""
+        while item_image_name[-1].isdigit():
+            item_id += item_image_name[-1]
+            item_image_name = item_image_name[:-1]
+
+        item_config = self.config["item"]["entities"][item_image_name]
```

Comment:
you don't use this until you create `chosen_asset` so move this just before to make it clearer where it is used